about.html

```diff
@@ -0,0 +1,121 @@
+        <li><a href=""><img>My page</a></li>
+        <li><a href=""><img>Log out</a></li>
+      </ul>
+    </div>
+    <div class="nav-bar">
+      <div class="burguer-button" id="open">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <a href="index.html"><img src="Capstone-img/logo-nfr.png" class="logo"></a>
+      <ul class="desk-ul">
+        <li><a href="about.html">About</a></li>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#events">Events</a></li>
+        <li><a href="#contact">Join</a></li>
+        <li><a href="#sponsor">Sponsor</a></li>
+        <li><a href="#contact">News</a></li>
+        <button class="soon">Soon!</button>
+      </ul>
+    </div>
+
+    <!-----------------------------Mobile menu-------------------------------->
+
+    <div class="mobile-menu" id="nav">
+
+      <a class="close-button" id="close"><img src="Capstone-img/closebotton.png"></a>
+      <ul class="mobile-ul">
+        <li><a href="index.html">About</a></li>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#events">Events</a></li>
+        <li><a href="#contact">Join</a></li>
+        <li><a href="#sponsor">Sponsor</a></li>
+        <li><a href="#contact">News</a></li>
+        <button class="soon">Soon!</button>
+      </ul>
+
+    </div>
+
+  </header>
+  <main>
+
+  <!------------------------------------Home page---------------------------------->
+  
+    <section  class="home-page">
+      <div class="home-page-content">
+        <p class="main-paragraph" id="main-paragraph-about">"Unforgettable moments in live rodeo and bull riding start right here"</p>
+        <h1 id="h1-about">Wrangler National Finals Rodeo</h1>
+        <p class="secundary-paragraf" id="secundary-paragraf-about">The first NFR was held in Dallas in 1959 and continued at that venue through 1961. From 1962 through 1964 Los Angeles hosted the competition. In 1964, however, Oklahoma City successfully bid to be the host city.</p>
+      </div>
+    </section>
+
+    <!-------------------------------------logo---------------------------------------->
+
+    <section class="logo-container">
+      <div class="logo-content">
+        <h2>Wrangler National Finals Rodeo Logo</h2>
+        <div class="red-line"></div>
+        <p>See our most recent logo for the 65 aniversary</p>
+        <img src="Capstone-img/logo-nfr.png">
+      </div>
+    </section>
+
+     <!-------------------------Bests--------------------------->
+
+     <section class="best-container">
+       <div class="best-content">
+       <h2>Top 2022 ranking</h2>
+       <div class="red-line"></div>
+       <p>The top 15 best cowboys of the year</p>
+       <div class="best-images">
+         <img src="Capstone-img/Saddle-Bronc-Riders-1064x482.jpg">
+         <img src="Capstone-img/Bareback-Riders-1.jpg">
+       </div>
+       
+      </div>
+
+    </section>
+     <!----------------------Sponsors---------------------->
```

Comment:
- [ ] `required`
- [ ] please make the photos horizontal like the original template
- [ ] please screenshots below

`your template`
![image](https://github.com/bohaz/capstone-project1/assets/66230306/fc4b5cfa-8e17-4432-a9bd-3bae94b73ce9)

`original template`
![image](https://github.com/bohaz/capstone-project1/assets/66230306/fa8e07f8-ae4d-413f-a247-ad72243830fa)